.github/workflows/cmake.yml

```diff
@@ -203,6 +203,16 @@ jobs:
       # This is so that device binaries can find the sycl runtime library
       run: cmake --build ${{github.workspace}}/build -j $(nproc)
 
+    - name: Information about platform
+      if: always()
+      run: |
+        export PATH=${workspace}/dpcpp_compiler/bin:$PATH
+        export CPATH=${workspace}/dpcpp_compiler/include:$CPATH
```

Comment:
are these exports for `sycl-ls` command? you can then just use them inline with the command itself (meaning, not setting exports) 😉